packages/core/data-transfer/src/engine/index.ts

```diff
@@ -372,11 +372,12 @@ class TransferEngine<
               }
 
               if (diff.kind === 'modified') {
-                return `Schema value changed at "${path}": "${diff.values[0]}" (${diff.types[0]}) => "${diff.values[1]}" (${diff.types[1]})`;
-              }
+                // eslint-disable-next-line eqeqeq
+                if (diff.values[0] == diff.values[1]) {
```

Comment:
I guess you meant `!=` instead of `==`? :eyes:

Reply:
Also, it means that if we have different values + different types, we would start by saying that we have different types instead of different values.

Reply:
Well I guess you don't want to swap the comparison but instead just invert the two messages